by-language/python-sqlalchemy/README.rst

```diff
@@ -0,0 +1,50 @@
+.. highlight: console
+
+############################################################
+Connect to CrateDB and CrateDB Cloud using Python/SQLAlchemy
+############################################################
+
+
+*****
+About
+*****
+
+Example programs demonstrating CrateDB's SQLAlchemy adapter and dialect.
```

Comment:
Currently, it is mostly about pandas/Dask operations through SQLAlchemy. This is a special topic, where we also added extensive documentation about, at [^1]. Here, we surely want to link to it.

[^1]: https://cratedb.com/docs/python/en/latest/by-example/sqlalchemy/dataframe.html

Reply:
The README has been improved, see its rendered version here.

-- https://github.com/crate/cratedb-examples/blob/amo/python-sqlalchemy/by-language/python-sqlalchemy/README.rst